Update README.md (#8172)
DataDog/integrations-core: windows_service/README.md

| Paused           | WARNING               |
| Unknown          | UNKNOWN               |

> If the service cannot be accessed by the Agent either due to permission restriction or the name is incorrect, the service check will report `UNKNOWN`.

Reviewer: ```suggestion
If the service cannot be accessed by the Agent due to a permission restriction or an incorrect name, the service check reports `UNKNOWN`.
```
